src/hm/hmInverter.h

```diff
@@ -849,7 +855,7 @@ class Inverter {
         uint8_t  mGetLossInterval = 0;  // request iv every AHOY_GET_LOSS_INTERVAL RealTimeRunData_Debug
         uint16_t mIvRxCnt  = 0;
         uint16_t mIvTxCnt  = 0;
-
+        uint16_t mAckCount = 0;
```

Comment:
Cool, gibt es die auch in der Statistik dann ?

Reply:
Fände ich cool, würde dann aber gleich mit anregen, die "temporären Daten" (loss rates) dann auch optisch etwas abzusetzen von den Gesamt-Statistik-Daten.

Es gibt aber leider nur sehr wenige User, die sowas überhaupt gesehen haben...

Reply:
Sollte dieser Ack Count schon in der 5801 sichtbar sein ?